python_modules/dagster/dagster_tests/daemon_tests/test_backfill.py

```diff
@@ -2383,75 +2387,6 @@ def test_asset_backfill_logging(caplog, instance, workspace_context):
     assert "AssetBackfillData" in logs
 
 
-def test_asset_backfill_asset_graph_out_of_sync_with_workspace(
```

Comment:
Pretty sure we can delete this since it is a test of what happens when the base asset job changes under your nose, which can no longer happen.